### PR TITLE
Shows cites in a tableView from cloud response

### PR DESCRIPTION
**Pull request by BenEmdon**
#### View hierarchy:
```
UIWindow
–UINavigationController
––[WelcomeViewController]
```
#### Changes:
* Data is now strongly typed and non optional
* TableView implementation
* Asynchronously fetches and sets layout
* _**swag splash screen**_

#### Screenshot:
screen shot 2016-06-20 at 9 14 05 pm

@TamimiGitHub @JackyChiu

### CODE REVIEW

File: TravelNPlay/TravelNPlay/WelcomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+//
+
+import UIKit
+import Parse
+
+class WelcomeViewController: UITableViewController {
+    
+    var cities = [String]()
+    
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) {
```
Comment (JackyChiu) [inline TravelNPlay/TravelNPlay/WelcomeViewController.swift]: Could you explain the override init and required init?

File: TravelNPlay/TravelNPlay/WelcomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+        self.title = "Cities"
+    }
+
```
Comment (JackyChiu) [inline TravelNPlay/TravelNPlay/WelcomeViewController.swift]: Could you explain the override init and required init?

Comment (BenEmdon) [inline TravelNPlay/TravelNPlay/WelcomeViewController.swift]: `override` is when your sub class wants to re define the initializer
`required` is when your subclasses sub classes are forces to implement that `init` definition

**Comment by JackyChiu** [comment]
Looks good 👍

**Pull request merged**